.github/workflows/coverage-pages.yaml

```
@@ -97,13 +97,19 @@ jobs:
           path: coverage/lcov-html/coverage-badge.svg
 
       - name: Upload Pages artifact (only on push to master in this repo)
+ ci/workflows
+        if: github.event_name == 'push' && github.ref == 'refs/heads/master' && github.repository == 'axel-download-accelerator/axel'
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 101: mapping values are not allowed in this context

(syntax-check)